src/clib/xtg/grd3d_roff2xtgeo_splitenz.c

```diff
@@ -0,0 +1,98 @@
+/*
+ ***************************************************************************************
+ *
+ * NAME:
+ *    grd3d_roff2xtgeo_zcorn.c
+ *
+ * DESCRIPTION:
+ *    Convert from ROFF arrays to XTGeo arrays: ZCORN
+ *
+ * ARGUMENTS:
+ *    nx, ny, nz       i     NLAY dimens
+ *    zoffset          i     Offsets in XYZ spesified in ROFF
+ *    zscale           i     Scaling in XYZ spesified in ROFF
+ *    splitenz         i     Split node vector
+ *    zdata            i     Input zdata array ROFF fmt
+ *    coordsv          o     Output zcorn array XTGEO fmt
+ *
+ * RETURNS:
+ *    Function: 0: upon success. If problems:
+ *              -1: Unsupported split type, supported types is 1 and 4
+ *              -2: Incorrect size of splitenz
+ *              -3: Incorrect size of zdata
+ *              -4: Incorrect size of zcorn
+ *
+ * TODO/ISSUES/BUGS:
+ *
+ * LICENCE:
+ *    cf. XTGeo LICENSE
+ ***************************************************************************************
+ */
+
+#include "libxtg.h"
+#include "libxtg_.h"
+#include "logger.h"
+
+int
+grd3d_roff2xtgeo_splitenz(
+                       int nz,
+                       float zoffset,
+                       float zscale,
```

Comment:
Sidenote, I use clang-format which will reformat this file. The format file is `src/clib/xtg/.clang-format`. In vscode the extension Clang-format (1.9.0) will assist in autoformatting

Reply:
I ran clang-format :)